src/Microsoft.AspNetCore.Razor.Tools/project.json

```diff
@@ -41,6 +41,7 @@
           "type": "platform",
           "version": "1.0.0"
         },
+        "System.Runtime.InteropServices.RuntimeInformation": "4.3.0-*",
```

Comment:
When you had your primitives 4.3 was this needed? Just curious so I know where this is coming in from.